wiki/Operator-101.md

```diff
@@ -0,0 +1,124 @@
+config1-config-statefulset-0                  10/10   Running            0          38m
+config1-config-statefulset-1                  10/10   Running            0          38m
+config1-config-statefulset-2                  10/10   Running            0          39m
+contrail-operator-dd5bb5c-klqwb               1/1     Running            0          42m
+control1-control-statefulset-0                4/4     Running            0          30m
+control1-control-statefulset-1                4/4     Running            0          30m
+control1-control-statefulset-2                4/4     Running            0          30m
+kubemanager1-kubemanager-statefulset-0        2/2     Running            0          30m
+kubemanager1-kubemanager-statefulset-1        2/2     Running            0          30m
+kubemanager1-kubemanager-statefulset-2        2/2     Running            0          30m
+provmanager1-provisionmanager-statefulset-0   1/1     Running            0          30m
+rabbitmq1-rabbitmq-statefulset-0              1/1     Running            0          39m
+rabbitmq1-rabbitmq-statefulset-1              1/1     Running            0          39m
+rabbitmq1-rabbitmq-statefulset-2              1/1     Running            0          39m
+vroutermasternodes-vrouter-daemonset-rgl4t    1/1     Running            0          28m
+vroutermasternodes-vrouter-daemonset-ttc7c    1/1     Running            0          28m
+vroutermasternodes-vrouter-daemonset-wn6qg    1/1     Running            0          28m
+vrouterworkernodes-vrouter-daemonset-gs4bw    1/1     Running            0          5m
+vrouterworkernodes-vrouter-daemonset-p7zkw    1/1     Running            0          5m
+vrouterworkernodes-vrouter-daemonset-pqfw9    1/1     Running            0          5m
+webui1-webui-statefulset-0                    3/3     Running            0          30m
+webui1-webui-statefulset-1                    3/3     Running            0          30m
+webui1-webui-statefulset-2                    3/3     Running            0          30m
+zookeeper1-zookeeper-statefulset-0            1/1     Running            0          8m
+zookeeper1-zookeeper-statefulset-1            1/1     Running            0          8m
+zookeeper1-zookeeper-statefulset-2            1/1     Running            0          8m
+```
+That's just one resource type which all te custom Contrail resources create in cluster during deployment.<br/>
+Below graph shows all resources created by Contrail during deployment with just one simple definition of single Manager custom resource.
+
+![operator-components](images/operator_components.png)
+
+## Miscellaneous elements of deployment
+
+Deployment described in previous section is just a core solution of contrail-operator.<br/>
+However, for different platforms, deployment may vary.<br/>
+Because of that in this repository are defined additional components which allow to deploy operatorized Contrail with platforms like Openstack or Openshift.
```

Comment:
Because of that, additional components are defined in this repository. They allow to deploy ...